azure-pipelines.yml

```diff
@@ -95,24 +96,48 @@ stages:
           secureFile: SonarSourceSecret.snk
 
       - task: DownloadSecureFile@1
-        displayName: 'Download pfx file'
-        name: pfx
+        displayName: 'Download p12 file'
+        name: SM_CLIENT_CERT
         inputs:
-          secureFile: 'SonarSource-2021-2023.pfx'
+          secureFile: digicert_authentication_certificate.p12
+ 
+      - task: DownloadSecureFile@1
+        displayName: 'Download crt file'
+        name: SM_CLIENT_CRT
+        inputs:
+          secureFile: cert_525594307.crt
+
+      - task: PowerShell@2
+        displayName: "Signing certificate setup"
```

Comment:
I think this tasks could benefit from a brief description of what we assume it does:

> Initialize the DigiCert Private Key Provider. What we think it does: The smctl tool authenticates with a client certificate (SM_CLIENT_CERT_FILE) and a client password (SM_CLIENT_CERT_PASSWORD). It uses an API Key (SM_API_KEY) and the ID of the certificate (SM_CERT) to check if the authenticated client is authorized to use the certificate specified and synchronize (potentially private) information about the certificate.

That is a lot of speculation but better than not having any information at all.